commands/release.go

```diff
@@ -195,6 +205,41 @@ func showRelease(cmd *Command, args *Args) {
 	os.Exit(0)
 }
 
+func downloadRelease(cmd *Command, args *Args) {
+	tagName := cmd.Arg(0)
+	if tagName == "" {
+		utils.Check(fmt.Errorf("Missing argument TAG"))
+	}
+
+	localRepo, err := github.LocalRepo()
+	utils.Check(err)
+
+	project, err := localRepo.MainProject()
+	utils.Check(err)
+
+	gh := github.NewClient(project.Host)
+
+	release, err := gh.FetchRelease(project, tagName)
+	utils.Check(err)
+
+	for _, asset := range release.Assets {
+		assetReader, apiError := gh.DownloadReleaseAsset(asset.ApiUrl)
+		utils.Check(apiError)
+		defer assetReader.Close()
```

Comment:
We could close the `assetReader` on each loop is finished instead of waiting for the whole program.